### DISCUSSION TITLE
关于使用文件工具转移后，电脑重启又要重新下载数据这件事情

### DISCUSSION THREAD

**Discussion by kamyo020704** [LocalizeLimbusCompany/LocalizeLimbusCompany#289] [问答 | Q&A]
根据https://www.zeroasso.top/docs/community/meitei/的内容，我将bat文件在D盘创建了一个文件夹用于储存边狱公司的缓存文件和音频文件。昨天使用时候并无问题，可是今天早上刚登入游戏时候提醒我要重新下载所有数据。贴吧看一些玩家即便没有使用这个工具的也会出现这样子的问题，而B站的https://t.bilibili.com/819946073292275748?tab=2评论区也有这样子的问题。是否有解决方法呢？又或者这是什么原因造成的呢？
![1](https://github.com/user-attachments/assets/73e2856f-2b83-4c62-8294-f088c155543e)
![2](https://github.com/user-attachments/assets/abb9b24b-19ea-4452-8396-d096f88be497)

**Comment by mucketguct** [comment]
您好，感谢反馈，这个问题有可能是一个月亮计划的程序bug啦，详情如下：

月亮计划的文件检索程序，有时会出现故障，如果遇到的是这一故障，会固定要求重新下载7.95G的数据（即当前游戏本体程序的容量总量）；

此bug的出现时机难以确定，有可能与使用文件工具转移文件无关；

一般来说，在遇到此bug后，只要重新下载一次这些数据，后续就永久不会重新遇到此bug啦；

目前我们同样遇到了不少次这一问题，目前也正在对这一问题的成因继续研究中，如果未来有了进一步的发现，与更有效的解决方案的话，届时也会继续来回复的啦，总之也非常感谢反馈啦
